src/API/VotingAPI.jsx

```diff
@@ -4,6 +4,10 @@ const api = axios.create({
     baseURL: import.meta.env.VITE_API_URL,
 });
 
+const blockchainApi = axios.create({
```

Comment:
Maybe define this with the other axios api endpoint in the API folder if it still exists

Reply:
No. Does not exist